tracks/python/exercises/leap/mentoring.md

```diff
@@ -1,3 +1,24 @@
+### Most common first attempt
+
+This is the most common solution by beginners to the leap year problem.
+Nudge the student towards the more concise solution by letting them know
+they can replace the 3 IF and 4 return statements with zero IF statements and a single return.
+This is done by using the logical operators OR & AND.
+
+```python
+def leap_year(year):
+    if year % 4 == 0: 
+        if year % 100 == 0: 
+            if year % 400 == 0: 
```

Comment:
```suggestion
            if year % 400 == 0:  # This can be improved upon.
```

Reply:
How can this be improved upon?  I mean, if this can be improved upon, why is this not also mentioned at line 10 and 11 as they are identical in functionality to this statement.

Reply:
This was based on my previous suggestion when I thought the original suggestion was only to replace the third `if`.

Reply:
Can probably mark it as resolved then?

Reply:
Yes 😄